stochrare/dynamics/diffusion.py

```diff
@@ -159,6 +190,8 @@ def trajectory(self, x0, t0, **kwargs):
         x = [x0]
         dt = kwargs.get('dt', self.default_dt) # Time step
         time = kwargs.get('T', 10.0)   # Total integration time
+        if dt < 0:
+            time = -time
```

Comment:
These two lines should be removed. Negative time steps should not be allowed, they can only lead to confusion with stochastic dynamics.